cpu/efm32/Makefile

```
@@ -1,6 +1,26 @@
 MODULE = cpu
 
 # add a list of subdirectories, that should also be build
-DIRS = periph families/$(EFM32_FAMILY) $(RIOTCPU)/cortexm_common
+DIRS += periph
+DIRS += $(RIOTCPU)/cortexm_common
+
+ifneq (,$(filter cpu_efm32gg,$(USEMODULE)))
+	DIRS += families/efm32gg
```

```suggestion
  DIRS += families/efm32gg
```
here and below (seems like there are tabs instead of 2 spaces)